scripts/tatt

```diff
@@ -207,9 +207,12 @@ if myJob.packageList is not None and len(myJob.packageList) > 0:
         if myJob.type == "keyword":
             targetarch = '~' + targetarch
 
+        output = myJob.packageList
+
         for p in myJob.packageList:
             print("Found the following package atom : " + p.packageString())
             # check if the package already has the needed keywords
+            # TODO: Is this needed anymore? I imagine nattka won't emit empty ones.
```

Comment:
Likely still needed for the local-file mode

Reply:
Agreed.